InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -41,6 +42,9 @@ def __init__(self, **kwargs: Any) -> None:
             pooling_type=GatedAttentionLayer.__name__,
             # average number of tiles is 56 for PANDA
             encoding_chunk_size=60,
+            cache_mode=CacheMode.MEMORY,
+            precache_location=CacheLocation.SAME,
```

Comment:
if we're fine-tuning what is the advantage of caching(save encoded features) and pre-caching(saving the all dataset up-front)?

Reply:
In case of finetuning, transform only includes loading tiles and not encoding the tiles. The caching will be of the tiles and not the features

Reply:
@vale-salvatelli to further clarify what @harshita-s means, here it's caching the loaded tiles into memory, so they don't need to be read from disk every time. IIRC this made an epoch drop from 30min to 5min in Harshita's experiments